woocommerce/woocommerce
# Change plugin name to "Woo Subscriptions" in release and daily testing workflows

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





"WooCommerce Subscriptions" recently changed to "Woo Subscriptions" which caused the plugin tests for it to fail in our daily and release testing workflows. This PR updates the two workflows to use the new plugin name.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to [Smoke test release](https://github.com/woocommerce/woocommerce/actions/workflows/smoke-test-release.yml)
2. In the "Run workflow" menu, select this branch.
3. Enter any recent `8.2.x` release version in the `WooCommerce Release Tag` field.
4. Click the green "Run workflow" button to start the run.
5. Wait for it to finish.
6. Verify that the "With Woo Subscriptions" job had run all the ~200 E2E tests, regardless of whether they passed or failed.
7. Verify that the other "With {PLUGIN NAME}" jobs had run all the ~200 E2E tests, regardless of whether they passed or failed.
8. Go to [Smoke test daily](https://github.com/woocommerce/woocommerce/actions/workflows/smoke-test-daily.yml).
2. In the "Run workflow" menu, select this branch.
1. Click the green "Run workflow" button to start the run.
6. Verify that the "Smoke tests on trunk with Woo Subscriptions plugin installed" job had run all the ~200 E2E tests, regardless of whether they passed or failed.
7. Verify that the other "Smoke tests on trunk with {PLUGIN NAME}" jobs had run all the ~200 E2E tests, regardless of whether they passed or failed.




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
